docs/src/almalinux_instructions.rst

```diff
@@ -0,0 +1,244 @@
+    sudo systemctl enable mongod
+
+    # Check that it is working
+    sudo systemctl status mongod
+
+.. note::
+
+    On RHEL7/CentOS7, MongoDB v14 can be installed as follows:
+    ``curl -fsSL https://rpm.nodesource.com/setup_14.x | bash - && yum install -y nodejs``
+
+
+3. Install carta-controller
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The easiest way to install carta-controller is using ``npm``. 
+
+.. code-block:: shell
+
+    sudo dnf install -y python3 make gcc-c++
+    sudo npm install -g --unsafe-perm carta-controller
+
+.. note::
+
+    The carta-controller executable will be installed at ``/usr/local/lib/node_modules/carta-controller``.
+    The carta-frontend will be installed at ``/usr/local/lib/node_modules/carta-controller/node_modules/carta-frontend/build``.
+
+.. note::
+    
+    Do not pass the ``--unsafe-perm`` flag to ``npm`` if using a local install.
+
+.. note::
+    Beta versions of carta-controller can be installed. For example, ``sudo npm install -g --unsafe-perm carta-controller-3.0.0-beta.1d``. 
+    Available versions can be found `here <https://www.npmjs.com/package/carta-controller>`_.
+
+.. note::
+    
+    On RHEL7/CentOS7 the carta-controller package can not run with the default gcc version 4.8.5 (there would be an error due to ``node-linux-pam``). 
+    A work around is to install a newer GCC version from source in order to get a newer ``libstdc++.so.6``, then add the location of the newer 
+    ``libstdc++.so.6`` to the LD_LIBRARY_PATH. After that, carta-controller can run on RHEL7/CentOS7.
+
+
+4. Install the carta-backend component
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The easiest way may be to install the carta-backend is from our cartavis RPM repository.
+
+.. code-block:: shell
+
+    # Install carta-backend
+    sudo curl https://packages.cartavis.org/cartavis-el8.repo --output /etc/yum.repos.d/cartavis.repo
+    sudo dnf -y install 'dnf-command(config-manager)'
+    sudo dnf -y install epel-release
+    sudo dnf -y config-manager --set-enabled powertools
+    sudo dnf -y install carta-backend
+
+    # Check that the backend can run and is version 2.0.0
+    /usr/bin/carta_backend --version
+
+
+.. note::
+    
+    If you install the beta version of carta-controller, you need to install the beta version of the carta-backend ``sudo dnf -y install carta-backend-beta``.
+
+
+5. Install Nginx
+~~~~~~~~~~~~~~~~
+
+carta-controller requires a webserver. Here we use `NGINX <https://www.nginx.com/>`_, but Apache should work too.
+
+.. code-block:: shell
+
+    # Install nginx:
+    sudo dnf install -y nginx
+    sudo systemctl start nginx
+    sudo systemctl enable nginx
+    sudo setsebool -P httpd_can_network_connect 1
+    sudo firewall-cmd --permanent --zone=public --add-service=http
+    sudo firewall-cmd --permanent --zone=public --add-service=https
+    sudo firewall-cmd --reload
+
+    # Generate private/public keys (optional if you do not already have SSL certificates):
```

Comment:
the private/public keys in `/etc/carta` are not related to nginx. They're used by carta-controller to sign/verify refresh and access tokens. Rather move this section to the controller configuration part